static/drink-list/drink-list.ts

```diff
@@ -3,13 +3,19 @@ class DrinkListCtrl {
   state: DrinkListStateService;
   db: Recipe[] = [];
   slugify: (name: string)=>string;
+  orderStatusService: OrderStatusService;
 
-  constructor(private $scope: angular.IScope,
+  constructor(private OrderStatusService: OrderStatusService,
+              private $scope: angular.IScope,
               private $location: angular.ILocationService,
               private $timeout: angular.ITimeoutService,
               private $window: angular.IWindowService,
               DrinkListStateService: DrinkListStateService,
               DrinksService: DrinksService) {
+    this.orderStatusService = OrderStatusService;
```

Comment:
OrderStatusService is already being copied to a local variable with the same name as the argument; that's what the "private" means in the constructor parameter list above.